test/svg-sprite.js

```diff
@@ -390,13 +400,10 @@ describe('svg-sprite', function () {
 
                     capturePuppeteer(preview, previewImage, function (error) {
                         should(error).not.ok;
-                        imageDiff({
-                            actualImage: previewImage,
-                            expectedImage: path.join(__dirname, 'expected', 'png', 'css.html.png'),
-                            diffImage: path.join(__dirname, '..', 'tmp', 'css', 'png', 'css.html.diff.png')
-                        }, function (error, imagesAreSame) {
+                        looksSame(previewImage,path.join(__dirname, 'expected', 'png', 'css.html.png'),
+                        function (error, r) {
```

Comment:
what is `r`?

Reply:
Maybe @bung87 can answer this?

Reply:
I think "r" is short for "result". I'll improve..